qiskit_experiments/library/randomized_benchmarking/clifford_utils.py

```diff
@@ -142,16 +132,7 @@ def _clifford_1q_int_to_instruction(
 def _clifford_2q_int_to_instruction(
```

Comment:
Yes, I agree. `CliffordUtils` can be promoted to an actual generator, but these function can be excluded if not intended to be used as a part of generator (probably this one can be `CliffordUtils` method, because it takes basis gates and likely the basis gates are instance variable of the proper RB circuit generator).